docs/design/custom-tab-placement.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Position a custom tab on the ribbon
+description: 'Learn how to control where a custom tab appears on the Office ribbon and whether it has focus by default.'
+ms.date: 11/01/2020
+localization_priority: Normal
+---
+
+
+# Position a custom tab on the ribbon (preview)
+
+You can specify where you want your add-in's custom tab to appear on the Office application's ribbon by using markup in the add-in's manifest.
+
+> [!NOTE]
+> This article assumes that you are familiar with the article [Basic concepts for add-in commands](add-in-commands.md). Please review it if you have not done so recently.
+
+> [!IMPORTANT]
+>
+> - The add-in feature and markup described in this article is in preview and is *only available in PowerPoint on the web*. We recommend that you try the markup out in test and development environments only. Do not use preview markup in a production environment or within business-critical documents.
+> - The markup described in this article only works on platforms that support requirement set **AddinCommands 1.3**. See [Behavior on unsupported platforms](#behavior-on-unsupported-platforms) below.
+
+You specify where you want a custom tab to appear by identifying which built-in Office tab you want it to be next to and specifying whether it should be on the left or right side of the built-in tab. You make these specifications by including either an [InsertBefore](../reference/manifest/customtab.md#insertbefore) (left) or an [InsertAfter](../reference/manifest/customtab.md#insertafter) element in the [CustomTab](../reference/manifest/customtab.md) (right) element in your add-in's manifest. (You cannot have both elements.)
+
+In the following example, the custom tab is configured to appear *just after* the **Review** tab. Note that the value of the `<InsertAfter>` element is the ID of the built-in Office tab. 
+
+```xml
+<ExtensionPoint xsi:type="ContosoRibbonTab">
+  <CustomTab id="TabCustom1">
+    <Group id="myCustomTab.group1">
+       <!-- additional markup omitted -->
+    </Group>
+    <Label resid="customTabLabel1" />
+    <InsertAfter>TabReview</InsertAfter>
```

Comment:
Is there somewhere that lists the available values for tab names?

Reply:
Not yet, but there will be. Probably before this goes public. I'll update as needed when that happens.